pom.xml

```diff
@@ -27,9 +27,59 @@
 
   <properties>
     <cdap.version>[6.7.0-SNAPSHOT,7.0.0)</cdap.version>
```

Comment:
Change this to 6.7.0-SNAPSHOT, the range may have problem while pulling the dependency.